api/src/org/labkey/api/data/SchemaTableInfo.java

```
@@ -857,6 +857,13 @@ public Domain getDomain()
         return null;
     }
 
+    @Nullable
+    @Override
+    public Domain getDomain(boolean forUpdate)
+    {
+        return getDomain();
```

Maybe this should throw `UnsupportedOperationException` when passed true?

---

Done.